openapi/components/parameters/ccdParam.yaml

```diff
@@ -0,0 +1,8 @@
+name: ccd
```

Comment:
I think we should avoid abbreviating these parameters. As time passes, we'll add more domains and spend more time context switching; it will be hard to remember `ccd` and `cd` by looking at them. Writing them out as `cityCouncilDistrict` and `communityDistrict` will make it easier to understand them at a glance.

Reply:
I disagree, for three reasons:
1) We consistently use the abbreviations `nta`, `cdta`, `cd`, and `ccd` throughout our projects, so it does not cause issues with context switching.
2) There is value in having shorter urls.
3) I believe we should be consistent in using the full names of variables when using full values, and acronyms for variable names when we are using shortened values.  For example:
 - `borough` should be used for "Manhattan", while `boro` should be used when the value is "MN" or "3"
 - `communityDistrict` should be used when the value is "Manhattan 11", and `cd` should be used when the value is "MN11" or "301"

Reply:
Let's prefer writing out params over shortening or abbreviating, with exceptions for certain tokens like `Id`. I think it makes the API more readable at a glance and I've been on projects in the past where you burn cycles as a new developer remembering how past devs decided to abbreviate things. We'll likely run into situations in the future where writing out becomes impractical but preferring writing out is a good default best practice and I think it makes sense for the situation here.

As for the `borough` use case, I would probably expect something like `boroughId` or `boroughCode`? That way we're being explicit about what property of the borough we're using for the look up, just like with `cityCouncilDistrictId`, etc.